duckduckgo/iOS
# Alpha build readiness

Task/Issue URL: https://app.asana.com/0/0/1205134602520178/f
Tech Design URL: https://app.asana.com/0/481882893211075/1204986042798431/f

**Description**:

In order to distribute to a separate app ID and to enable in-development features (e.g the debug menu) for alpha builds, we should create a new build configuration in the iOS project.

This also includes icons given to me by design.

Note that this configuration essentially has debug features on (e.g the Debug menu, compiler flagger features) but has a release configuration which goes to our dedicated DuckDuckGo-Alpha app on AppStore Connect (so no danger of any of these builds actually getting released).

**Steps to test this PR**:
1. CI checks should be enough. It just adds a new configuration and sets that up...
2. However, if you want more proof, you can check the PR task that the AdHoc (another type of release build) worked https://app.asana.com/0/0/1205266414152146/f (I built one to be attached it to that task)
3. Check AppStore Connect -> Apps -> DuckDuckGo-Alpha -> TestFlight and see that there’s a build there that I created using this branch.

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
